Makefile

```diff
@@ -22,12 +27,14 @@ third_party/make-env/conda.mk:
 include third_party/make-env/conda.mk
 
 env:: | $(CONDA_ENV_PYTHON)
-	git submodule init
-	git submodule update --init --recursive
-	mkdir -p env/symbiflow
+	mkdir -p env/${TOOLCHAIN}
 	wget -O ${SYMBIFLOW_ARCHIVE} ${SYMBIFLOW_URL}
-	tar -xf ${SYMBIFLOW_ARCHIVE} -C env/symbiflow
+	tar -xf ${SYMBIFLOW_ARCHIVE} -C env/${TOOLCHAIN}
 	rm ${SYMBIFLOW_ARCHIVE}
+ifeq ("${TOOLCHAIN}", "quicklogic")
+	cp -r env/conda/envs/quicklogic/share/yosys/* env/conda/envs/quicklogic/share/
+	sed -i "s/\/home\/kkumar\/symbiflow-arch-defs\/build\/env\/conda\/bin\/python3/\/usr\/bin\/env python3/" env/${TOOLCHAIN}/install/bin/python/qlfasm #fix path to interpreter, see https://github.com/QuickLogic-Corp/quicklogic-fpga-toolchain/issues/16
+endif
```

Comment:
This has two patches, one related to a fixed path (https://github.com/QuickLogic-Corp/quicklogic-fpga-toolchain/issues/16), and the other to a yosys issue for which the share directory cannot be found.

It would be better to have a FIXME or TODO before this block of code explaining why it is necessary

Reply:
done